test/runtime/fqdn.go

```diff
@@ -122,9 +126,9 @@ var _ = Describe("RuntimeFQDNPolicies", func() {
 		}
 
 		ciliumOutisdeImages = map[string]string{
-			WorldHttpd1: helpers.HttpdImage,
-			WorldHttpd2: helpers.HttpdImage,
-			WorldHttpd3: helpers.HttpdImage,
+			OutsideHttpd1: helpers.HttpdImage,
```

Comment:
Why the rename? I thought the world* IPs were already "outside" our network

Reply:
Because the OutsideHttpd1 server was not created at all. The name of the container was already pick.